horovod/mxnet/mpi_ops.cc

```diff
@@ -38,6 +38,8 @@ std::string GetOpName(std::string prefix, char* name) {
 }
 } // namespace
 
+const auto EXEC_CTX = Context::CPU();
```

Comment:
nit: use static const since this variable is only for MXNet; maybe also name it MX_EXEC_CTX?

Reply:
MX_EXEC_CTX sounds good. Let's do it in my next PR if this is the only concern.

Reply:
SGTM.